src/common/opencl.c

```diff
@@ -930,7 +930,7 @@ static gboolean _opencl_device_init(dt_opencl_t *cl,
            "%s" G_DIR_SEPARATOR_S "programs.conf", kerneldir);
 
   char *escapedkerneldir = NULL;
-#ifndef __APPLE__
+#if !defined(__APPLE__) && !(defined(__linux__) && defined(__aarch64__))
```

Comment:
I guess this ideally needs testing on Windows ARM64 as well...

Reply:
Sure, but at least this is a step forward. We'll see if someone with such a configuration has issue and we will fix at this point.